### PR TITLE
[BLOGS-1369] Helping QA automation

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/BLOGS-1369

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Is there some sort of logic to the labelling?

**Comment by benolot** [comment]
Discussed with testers and these are the places they wanted it labelled

File: templates/blog/show.html.twig [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 {% if loop.index0 == 3 %}
                     {% set charLimit = 600 %}
                 {% endif %}
-                <li class="post br-keyline">
+                <li id="blogs-post-{{ loop.index }}" class="post br-keyline">
```
Comment (votemike) [inline templates/blog/show.html.twig]: Could this also be a data-attribute?

**Pull request merged**